test/feature/Yield/TryCatchGenerator.js

```diff
@@ -1,3 +1,4 @@
+// Options: --symbols
```

Comment:
This was an experiment to confirm my understanding. (generators depends upon `Symbol.iterator`).  Removed.